go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/deckarep/golang-set v1.7.1
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/droptheplot/abcgo v0.0.0-20171120220436-23529565504c // indirect
+	github.com/fzipp/gocyclo v0.0.0-20150627053110-6acd4345c835 // indirect
```

Comment:
Probably we need to add `GO111MODULE=off` to our linting scripts, like here https://github.com/RedHatInsights/insights-results-aggregator/blob/master/goconst.sh